### DISCUSSION TITLE
instance.__dict__ does not contain @hybrid_property in SQLAlchemy or when SQL model instance is directly as response

### DISCUSSION THREAD

**Discussion by rajatgng** [fastapi/fastapi#8718] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python

#SQLAlchemy Model
class User(Base):
    __tablename__ = "users"
    id = Column(Integer, primary_key=True, index=True, nullable=False)
    email = Column(String, unique=True, index=True, nullable=False)
    password = Column(String, nullable=False)
    created_at = Column(
        TIMESTAMP(timezone=True), nullable=False, server_default=text("now()")
    )
    posts= relationship("Post", back_populates="owner")
    
   
    @hybrid_property
    def posts_count(self):
        return len(self.posts)
    
   
    @posts_count.expression
    def posts_count(self):
        return len(self.posts)

#route path
@router.post("/login", response_model=schemas.LoggedInUserResDTO)
def login(
    payload: OAuth2PasswordRequestForm = Depends(), db: Session = Depends(get_db)
):

    user = crud.getUserByEmail(db, payload.username)
    print(user.posts_count)
    if not user or not utils.verify_password(payload.password, user.password):
        raise HTTPException(
            status_code=status.HTTP_403_FORBIDDEN,
            detail=f"Invalid Credentials",
        )
    auth_token = oauth2.createJWTToken({"user_id": user.id})
    res = user.__dict__
    res.update(
        {
            "access_token": auth_token,
            "token_type": "Bearer",
        }
    )
    return res

#also user. __dict__ does not contain "posts_count" property as well
```

### Description

When using @hybrid_property, that property doesn't contain in dict. Even when directly returning model response, even that doesn't contain the hybrid property.

In the below code, I am trying to access "posts_count" after storing dict value but it's doesn't present in it

### Operating System

Linux

### Operating System Details

Ubuntu 20

### FastAPI Version

FastAPI: 0.70.1

### Python Version

Python 3.8.10

**Comment by gordthompson** [comment]
Pydantic's "ORM mode" already supports hybrid properties.

https://github.com/tiangolo/fastapi/issues/218#issuecomment-503988741

**Comment by rajatgng** [comment]
> Pydantic's "ORM mode" already supports hybrid properties.
> 
> [#218 (comment)](https://github.com/tiangolo/fastapi/issues/218#issuecomment-503988741)

It is available when ORM Model is directly used, but when we convert that model to dict using `__dict__` method, the hybrid properties are not contained in it.

**Comment by aimfeld** [comment]
I'm trying to cache FastAPI responses using [fastapi-cache](https://github.com/long2ice/fastapi-cache), which uses FastAPI's `jsonable_encoder` internally. The cached response is missing hybrid properties, which I assume is related to this issue. Interestingly, [aggregated attributes](https://sqlalchemy-utils.readthedocs.io/en/latest/aggregates.html) are working fine with `jsonable_encoder`.
